llumnix/arg_utils.py

```diff
@@ -52,10 +52,11 @@ class EngineManagerArgs:
     profiling_result_file_path: str = ""
 
     gpu_type: str = "a10"
-
-    migration_backend: str = "rpc"
-    migration_cache_blocks: int = 512
-    last_stage_max_blocks: int = 4
+    migration_backend_init_timeout: float = 10.0
+    migration_backend: str = "gloo"
```

Comment:
If the default migration backend of Llumnix becomes gloo, we can change manager arguments fixed-node-init-instance to disable-fixed-node-init-instance, and init-instance-by-manager to disable-init-instance-by-manager, so that user can run correctly using default migration backend without setting fixed-node-init-instance and init-instance-by-manager.

Reply:
set it to nccl, now.

Reply:
No, nccl can not adapt to model parallelism model, so Llumnix's default migration backend should be gloo.